src/lucky_migrator/create_table_statement.cr

```diff
@@ -2,6 +2,9 @@ class LuckyMigrator::CreateTableStatement
   private getter rows = [] of String
   private getter index_statements = [] of String
 
+  alias ColumnType = String.class | Time.class | Int32.class | Int64.class | Bool.class | Float.class
+  alias ColumnDefaultType = String | Time | Int32 | Int64 | Bool | Symbol | Nil
```

Comment:
I think you can add `Float32 | Float64` here. That way you don't need to manually add `| Float` on line 76

Reply:
Of course! I get it now. The Float:Class is the type declaration, but the default type will be Float32 or Float64. I was actually really confused about this and didn't understand the difference